src/AddIns/Uno.UI.MSAL/Uno.UI.MSAL.Windows.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
 
 	<PropertyGroup>
-		<TargetFrameworks>uap10.0.19041</TargetFrameworks>
+		<TargetFrameworks>$(NetUWPOrWinUI)</TargetFrameworks>
```

Comment:
This was previously targeting uap only. Now it properly targets uap or `net7.0-windows10.0.19041.0` depending on whether we are building Uno.UI.MSAL or Uno.WinUI.MSAL.

However, the nuspecs are not updated yet. @jeromelaban Would you be able to do that? I was trying to mimic what Uno.UI does for including toolkit, but got confused by https://github.com/unoplatform/uno/issues/12823